commercetools-models/src/main/java/io/sphere/sdk/customers/queries/CustomerInStoreQueryBuilder.java

```diff
@@ -0,0 +1,133 @@
+package io.sphere.sdk.customers.queries;
+
+import io.sphere.sdk.customers.Customer;
+import io.sphere.sdk.customers.expansion.CustomerExpansionModel;
+import io.sphere.sdk.expansion.ExpansionPathContainer;
+import io.sphere.sdk.queries.QueryPredicate;
+import io.sphere.sdk.queries.QuerySort;
+import io.sphere.sdk.queries.ResourceMetaModelQueryDslBuilderImpl;
+
+import java.util.List;
+import java.util.function.Function;
+
+public final class CustomerInStoreQueryBuilder extends ResourceMetaModelQueryDslBuilderImpl<CustomerInStoreQueryBuilder, Customer, CustomerInStoreQuery, CustomerQueryModel, CustomerExpansionModel<Customer>> {
+
+    private CustomerInStoreQueryBuilder(final CustomerInStoreQuery template) {
+        super(template);
+    }
+
+    public static CustomerInStoreQueryBuilder of(final String storeKey) {
+        return new CustomerInStoreQueryBuilder(CustomerInStoreQuery.of(storeKey));
+    }
+
+    @Override
+    protected CustomerInStoreQueryBuilder getThis() {
+        return this;
+    }
+
+    @Override
+    public CustomerInStoreQuery build() {
+        return super.build();
+    }
+
+    @Override
+    public CustomerInStoreQueryBuilder fetchTotal(final boolean fetchTotal) {
+        return super.fetchTotal(fetchTotal);
+    }
+
+    @Override
+    public CustomerInStoreQueryBuilder limit(final Long limit) {
+        return super.limit(limit);
+    }
+
+    @Override
+    public CustomerInStoreQueryBuilder limit(final long limit) {
+        return super.limit(limit);
+    }
+
+    @Override
+    public CustomerInStoreQueryBuilder offset(final Long offset) {
+        return super.offset(offset);
+    }
+
+    @Override
+    public CustomerInStoreQueryBuilder offset(final long offset) {
```

Comment:
Why overwrite all the methods here, if it dosn't serve any purpose it becomes confusing.